app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughActivityPresenter.kt

```diff
@@ -20,24 +20,27 @@ class WalkthroughActivityPresenter @Inject constructor(
 ) : WalkthroughActivityListener {
   private lateinit var topicId: String
   private lateinit var binding: WalkthroughActivityBinding
+  private val mViewModel by lazy {
+    getWalkthroughViewModel()
+  }
 
   fun handleOnCreate() {
     binding = DataBindingUtil.setContentView(activity, R.layout.walkthrough_activity)
 
     binding.apply {
-      viewModel = getWalkthroughViewModel()
+      viewModel = mViewModel
```

Comment:
```viewModel = viewModel``` is causing some error because it is not recoginsing RHS ViewModel as global ViewModel we declared above.

Reply:
@rt4914 What are your views here? `mViewModel` isn't a good name as per our naming convention and `viewModel` and `walkthroughViewModel` are causing problem.

Reply:
You can also name this as `walkthroughActivityViewModel `